go/pools/id_pool_test.go

```diff
@@ -84,23 +72,16 @@ func TestIDPoolGetFromUsedSet(t *testing.T) {
 	id1 := pool.Get()
 	pool.Get()
 	pool.Put(id1)
-
-	if got := pool.Get(); got != 1 {
-		t.Errorf("pool.Get() = %v, want 1", got)
-	}
-
+	got := pool.Get()
+	assert.EqualValues(t, uint32(1), got)
 	pool.want(&IDPool{used: map[uint32]bool{}, maxUsed: 2}, t)
 }
 
 func wantError(want string, t *testing.T) {
 	rec := recover()
-	if rec == nil {
-		t.Errorf("expected panic, but there wasn't one")
-	}
+	require.NotNil(t, rec, "expected panic, but there wasn't one")
 	err, ok := rec.(error)
-	if !ok || !strings.Contains(err.Error(), want) {
-		t.Errorf("wrong error, got '%v', want '%v'", err, want)
-	}
+	require.True(t, ok && strings.Contains(err.Error(), want), "wrong error, got '%v', want '%v'", err, want)
```

Comment:
Split to two checks. One is `require.True`, the other is `require.Contains`.
Also, what makes this a `require` rather than `assert`?